docs/Third-party-integrations/GCP-BigQuery-Integration/python/cloud_function.py

```
@@ -0,0 +1,99 @@
+
+    return (json.dumps({
+        "status": "ok",
+        "files_processed": len(csv_files),
+        "rows_loaded": rows_loaded,
+    }), 200, {"Content-Type": "application/json"})
+
+
+def download_export(download_url: str) -> bytes:
+    """Download the export ZIP file from the ABsmartly API."""
+    url = f"{ABSMARTLY_API_URL.rstrip('/')}{download_url}"
+    response = requests.get(url, headers={"Authorization": f"Api-Key {ABSMARTLY_API_KEY}"})
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)